Objects/stringlib/formatter.h

```diff
@@ -572,7 +572,7 @@ fill_number(STRINGLIB_CHAR *buf, const NumberFieldWidths *spec,
     if (spec->n_sign == 1) {
         *buf++ = spec->sign;
     }
-    if (spec->n_prefix) {
+    if (spec->n_prefix && prefix) {
```

Comment:
Is it possible that prefix is NULL?

Reply:
``format_float_internal()`` calls the fill function with prefix=NULL.